src/btree/bt_debug.c

```diff
@@ -926,7 +926,7 @@ __debug_tree_shape_worker(WT_DBG *ds, WT_REF *ref, int level)
           "%d %s\n",
           level * 3, " ", level, __debug_tree_shape_info(ref, buf, sizeof(buf))));
         WT_INTL_FOREACH_BEGIN (session, ref->page, walk) {
-            if (walk->state == WT_REF_MEM)
+            if (__wt_atomic_loadv8(&walk->state) == WT_REF_MEM)
```

Comment:
Weird suggestion but should we wrap ref->state in like a `WT_REF_STATE` macro? Also we could bury state under __.